registry-config.yaml

```diff
@@ -0,0 +1,17 @@
+# https://docs.docker.com/registry/configuration/
+
+version: 0.1
+log:
+  level: debug # info, warn, error 
+  fields:
+    service: registry
+storage:
+  cache:
+    blobdescriptor: inmemory
+  filesystem:
+    rootdirectory: /mnt/docker0storage/registry
+http:
+  addr: 443:5000 
+  host: docker.cdot.systems
+  headers:
+    X-Content-Type-Options: [nosniff]
```

Comment:
I think we need to add the following, but am unsure of the values provided.

``` yaml
auth:
  token:
    issuer: 'Acme auth server'
    realm: "https://docker.cdot.systems/auth"
    service: "Docker registry"
    rootcertbundle: /etc/letsencrypt/live/docker.cdot.systems
```

Reply:
Fix